### PR TITLE
Removed duplicate code

### PR DESCRIPTION
**Pull request by radarhere**

### CODE REVIEW

File: src/PIL/ImageDraw.py [hunk 1]
```diff
@@ -351,34 +351,20 @@ def floodfill(image, xy, value, border=None, thresh=0):
...
+        newedge = []
+        for (x, y) in edge:
+            for (s, t) in ((x+1, y), (x-1, y), (x, y+1), (x, y-1)):
+                try:
+                    p = pixel[s, t]
+                except IndexError:
+                    pass
+                else:
+                    if _color_diff(p, background) <= thresh if border is None \
+                       else (p != value and p != border):
```
[the remark this answers is not in the document]
Comment (radarhere) [reply src/PIL/ImageDraw.py]: Okay, sure, I've updated the commit. It’s just an ‘x if y else z’ ternary within an if statement.

[the remark this answers is not in the document]
Comment (radarhere) [reply src/PIL/ImageDraw.py]: So yes, your estimation of how it performs is correct, `y` being evaluated first, etc.

**Comment by hugovk** [comment]
This has a conflict now https://github.com/python-pillow/Pillow/pull/3326 is merged.

**Comment by radarhere** [comment]
Rebased

**Pull request merged**